jumble/src/iframe-ctx.ts

```diff
@@ -144,4 +144,11 @@ export const setupIframe = () =>
       console.log("onLLMRequest res", res);
       return res as any;
     },
+    async onReadWebpageRequest(_context: any, payload: string) {
+      console.log("onReadWebpageRequest", payload);
+      const url = payload.trim().replace(/^["']+|["']+$/g, "");
```

Comment:
This looks like due to the stringification of the payload object, which isn't necessary as it's a string initially

Reply:
Nice catch, fixed and removed.